charts/pega/values.yaml

```diff
@@ -80,22 +80,22 @@ global:
         targetPort: 8080
       nodeType: "WebUser"
       replicas: 1
-      # Enter any additional java options.
+      # Provide additional java options if any, like configuring timezone using -Duser.timezone
       javaOpts: ""
       # Set your Pega diagnostic credentials.
       pegaDiagnosticUser: ""
       pegaDiagnosticPassword: ""
       # When provider is eks, configure alb cookie duration seconds equal to passivation time of requestors
       alb_stickiness_lb_cookie_duration_seconds: 3660
```

Comment:
Should this be under `service`?

Reply:
alb_stickiness_lb_cookie_duration_seconds property gets added to ingress (annotations) object not to service, may be so it is not put in service section. 
It gets added to ingress only when provider is EKS, handled in the code.  So functionally should be fine having it for any other provider.
@Heshma  Please add further.

Reply:
alb_stickiness_lb_cookie_duration_seconds property gets added to ingress object. But it would be good to move it under service as we deploy ingress only if we have service section in a tier. Yes it gets added to ingress only when provider is EKS as @kishorv10 mentioned

Reply:
alb_stickiness_lb_cookie_duration_seconds property gets added to ingress object. But it would be good to move it under service as we deploy ingress only if we have service section in a tier. Yes it gets added to ingress only when provider is EKS as @kishorv10 mentioned

Reply:
Made these changes. Moved `alb_stickiness_lb_cookie_duration_seconds `  under `service `section. Also removed this property from batch tier configuration, as batch tier won't have a `service`